### PR TITLE
Made the website appear (with the loading)

### PR DESCRIPTION
**Pull request by gilemos**
Hey everyone! I did the website view. See how it looks like:

Video Walkthrough

### CODE REVIEW

File: Travel Scheduler/Helper Files/APIManager.m [hunk 1]
```diff
@@ -286,6 +286,30 @@ - (void)getPhotoFromReference:(NSString *)reference withCompletion:(void (^)(NSU
     [task resume];
 }
 
+- (void)getWebsiteLinkOfPlaceWithId:(NSString *)placeId withCompletion:(void (^)(NSString *placeWebsiteString, NSError *error))completion
```
Comment (aliu917) [inline Travel Scheduler/Helper Files/APIManager.m:289]: I'm just curious, but this seems to be the same API call as the getCompleteInfoOfLocationWithName and that API call also gives the website. Is there a reason why you made a new one and didn't just reuse that one?

File: Travel Scheduler/View Controllers/PlaceWebsiteViewController.m [hunk 2]
```diff
@@ -21,9 +21,22 @@ - (void)viewDidLoad {
...
     _webView = [[WKWebView alloc] initWithFrame:self.view.frame];
+    _webView.navigationDelegate = self;
+    self.spinner = [[UIActivityIndicatorView alloc] initWithActivityIndicatorStyle:UIActivityIndicatorViewStyleWhiteLarge];
+    [self.spinner setCenter:self.view.center];
+    self.spinner.backgroundColor = [UIColor lightGrayColor];
+    [self.spinner hidesWhenStopped];
+    [_webView addSubview:self.spinner];
+    [self.spinner startAnimating];
     [_webView loadRequest:request];
     _webView.frame = CGRectMake(self.view.frame.origin.x,self.view.frame.origin.y, self.view.frame.size.width, self.view.frame.size.height);
```
Comment (aliu917) [inline Travel Scheduler/View Controllers/PlaceWebsiteViewController.m:32]: You should set the frame in the viewWillLayoutSubviews.

**Review by aliu917 — APPROVED** [review]
Great work! The websites look really good.

File: Travel Scheduler/Helper Files/APIManager.m (hunk 1 above)
Comment (frankboamps) [reply Travel Scheduler/Helper Files/APIManager.m:289]: Yeah. I also thought we had already gotten the website url from the getCompleteInfoOfLocationWithName API call.

File: Travel Scheduler/Views/DetailHeaderCell.m [hunk 3]
```diff
@@ -323,11 +323,11 @@ - (void)customLayouts
...
 
 - (void)setSelected:(BOOL)selected animated:(BOOL)animated
-{
-    [super setSelected:selected animated:animated];
-    // Configure the view for the selected state
-}
-
+    {
+        [super setSelected:selected animated:animated];
+        // Configure the view for the selected state
```
Comment (frankboamps) [inline Travel Scheduler/Views/DetailHeaderCell.m]: Remove comment or make TO DO

**Review by frankboamps — APPROVED** [review]
On the whole it looks really good but I did not understand the reason for the new API call. Could you elaborate on that.

**Pull request merged**